app/assets/fonts/glyphs.txt

```diff
@@ -1 +1 @@
- !"#$%&'()+,-./0123456789:;>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz~ «»¿ÀÁÈÉÊÎÓÚàáâãçèéêëíîïñóôùúû ‑—‘’“”…‹中体文简
+ !"#$%&'()+,-./0123456789:;>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz~  «»¿ÀÁÈÉÊÎÓÚàáâãçèéêëíîïñóôùúû ‑—‘’“”…‹中体文简
```

Comment:
do we need to update `scripts/yaml_characters` to always hardcode nbsp in there?

Reply:
No, the lint failure is actually correct. The non-breaking space is actually in here twice.

Like Andrew described above, the issue was actually in the documented command for subsetting the fonts. It used a CLI option that caused the list of characters to be trimmed. I just force pushed a new commit to use a different option and updated the documentation.